text/0000-fabric-private-chaincode-lite.md

```diff
@@ -310,64 +310,72 @@ The registry is particularly relevant for clients, for retrieving an FPC chainco
 
-	Next, the Chaincode enclave participates in an attestation protocol.
-	This protocol may be different depending on the TEE technology used.
-	In the case of Intel&reg; SGX, the enclave produces a quote that allows to verify that a legitimate TEE is running a correct version of software and a particular FPC Chaincode (identified through MRENCLAVE).
-	This quote is then sent to the Intel&reg; Attestation Service (IAS) for validation.
-	The IAS returns an attestation report (evidence) that is used to proof that the enclave runs a particular chaincode.
+Following this deployment step, from a Fabric perspective, the FPC chaincode is operative.
+However, any FPC Chaincode query/invocation will return an error because the FPC chaincode enclave (and its cryptographic material) are still uninitialized. 
 
-	To complete the enclave creation, the attestation evidence and the enclave state including all the names, versions, cryptographic keys are encrypted using the enclave's sealing mechanism. 
-	The enclave returns the public keys, sealed state, and the attestation evidence to the FPC Shim (outside the enclave).
-	
-    The FPC Shim then stores the sealed state in its local storage provided by the external launcher.
-    This sealed state enables recreating the FPC Chaincode enclave and reprovisioning it with the private keys and correct state,
-    e.g., during restart of the Peer or because the external builder restarts the chaincode previously stopped due to idleness.
-    Note that the properties of the seal operation guarantee that only an enclave running the correct FPC Chaincode will be able to unseal that information.
+### Initialize and Register the FPC Chaincode enclave.
 
-	The Peer signs the enclave public keys for its organization, thereby taking ownership of the new Enclave for that organization. 
+**(FPC Lite makes the simplifying assumption that, for each deployed FPC package, a single enclave is initialized and registered.)**
 
-	Once the enclave is up and running, the `peer lifecycle chaincode createenclave` command invokes a register transaction for the newly created Chaincode Enclave.
-	The public parameters of the Chaincode Enclave along with the attestation evidence is sent to the FPC Registry, which validates them internally by checking consistency and signatures, and then stores the new entry.
-	The register invocation is invoked at a sufficient set of endorsing peers to satisfy the FPC Registry endorsement policy.
-	Finally, a Register transaction is submitted to the Ordering Service.
-	By committing a Register transactions, the FPC Chaincode Attestation is now visible proof of validity of the FPC Chaincode to all participants in the Channel.
+The administrator of the peer hosting the FPC chaincode enclave is responsible for the initialization and registration of the enclave.
+The operation is performed by executing the `initEnclave` admin command.
+
+The command can be triggered through the FPC Client SDK.
```

Comment:
```suggestion
The command can be triggered through the FPC Client SDK (see [earlier Deployment Section](#deployment)
```

Reply:
Oops, too quick in pressing button, here now also the overall comment.  Looks overall good, just that the description of `initEnclave` admin command doesn't match the spec ...